### PR TITLE
Add interactive capability for show command

### LINKED ISSUE #233: interactive cli show command
For DX we can make the show command more interactive. If no argument is passed, the aggregate is asked it for first. Here we can display a list of aggregates from which he can select one. Then you will be asked for the aggregate ID. 
This should only happen as an option if no arguments is passed, otherwise it will use the arguments.

### PR DESCRIPTION
**Pull request by icanhazstring**
This should resolve #233

### CODE REVIEW

File: src/Console/Command/ShowCommand.php [hunk 1]
```diff
@@ -40,16 +43,30 @@ public function __construct(Store $store, EventSerializer $serializer, Aggregate
...
     }
 
     protected function execute(InputInterface $input, OutputInterface $output): int
     {
         $console = new OutputStyle($input, $output);
 
-        $aggregate = InputHelper::string($input->getArgument('aggregate'));
-        $id = InputHelper::string($input->getArgument('id'));
+        $aggregate = InputHelper::nullableString($input->getArgument('aggregate'));
+        while ($aggregate === null) {
```
Comment (DanielBadura) [inline src/Console/Command/ShowCommand.php]: Isnt an `if` sufficient for both arguments?

Comment (icanhazstring) [reply src/Console/Command/ShowCommand.php]: If you hit just enter you will be asked again until you cancel the command, or enter something.

Comment (DanielBadura) [reply src/Console/Command/ShowCommand.php]: I'm not sure if this approach or a error message which hints the user that he needs to choose one option to proceed is better DX.

But lets keep it as is with the requestioning until a value is choosen, maybe with an additional text hinting the user that no input was given? This is up to you if you want to put more effort into it. 

A Testcase covering that the arguments are now optional would be great.

Comment (icanhazstring) [reply src/Console/Command/ShowCommand.php]: Good point, will add a hint and test 😉

**Comment by icanhazstring** [comment]
I removed the `while` statement as its not necessary. The question helper will do its part here.
Also added tests using `CommandTester`. If this is not wanted I can refactor it to be structured as the rest of the tests :)

**Review by DanielBadura — APPROVED** [review]
This looks good, thank you @icanhazstring!

**Pull request merged**